cmd/image/import/import.go

```diff
@@ -262,20 +262,20 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --object rhel-83-1003
 			return err
 		}
 		start := time.Now()
-		err = utils.PollUntil(time.Tick(2*time.Minute), time.After(opt.WatchTimeout), func() (bool, error) {
+		var message string
```

Comment:
Will making this variable available to the block below work?

Reply:
This variable is accessed and modified outside the block as well

Reply:
Sure Manju, I'll take a look at it.

Reply:
Hi @mkumatag, I see that it's reused here: 
https://github.com/ppc64le-cloud/pvsadm/pull/680/files#diff-ae8fe9bef8e5a7ec3362a0d4dca4f3e432f4b1fe64e7c1dde9b23d98dbc8f937R308-R309

Or, we could switch to short-hand declaration in both cases. Just a thought..

Reply:
why are we reusing it? are we sharing anything between those 2 blocks? if not - lets define locally.

Reply:
Added short-hand declaration, since we aren't sharing anything between the 2 blocks